content-src/components/StartupOverlay/StartupOverlay.jsx

```diff
@@ -15,8 +15,29 @@ export class _StartupOverlay extends React.PureComponent {
 
     this.state = {
       emailInput: "",
-      overlayRemoved: false
+      overlayRemoved: false,
+      flowId: "",
+      flowBeginTime: ""
```

Comment:
this should probably be a 0, since it's a timestamp. let's keep the "types" consistent